Arrays/Arrays.js

```diff
@@ -0,0 +1,39 @@
+/*
+Array is a type of data structure. It is used for storing data and can contain or store a large number of data that has a similar data type.
+
+Each item in an array is called an element.
+The total number of elements in an array is called length. Counting an array starts from zero.
+Each element also has its own index, which is used to access the element
+There are different operations that can be performed when using an array. They include; 
+
+Traverse − print all the array elements one by one.
+Insertion − Adds an element at the given index.
+Deletion − Deletes an element at the given index.
+Search − Searches an element using the given index or by the value.
+Update − Updates an element at the given index.
+*/
```

Comment:
Nice summary but note that array can store complex data structure and mixed data structure e.g `const arr = [1, "boy", {}];`.

Read more about Array [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array)

Reply:
okay